docs/static/arcsight-module.asciidoc

```diff
@@ -104,32 +100,19 @@ the EB event stream.
 
 image::static/images/arcsight-diagram-adp.svg[ArcSight Event Broker architecture]
 
-[[arcsight-requirements-eventbroker]]
-===== Requirements
-
-* These instructions assume you have the Elastic Stack (Logstash, Elasticsearch,
-Kibana) already installed. The products you need are
-https://www.elastic.co/downloads[available to download] and easy to install. The
-Elastic Stack 5.6 or higher is required for this module.
-* The Elastic Stack is running locally with default ports exposed, namely
-Elasticsearch as "localhost:9200" and Kibana as "localhost:5601".
-* By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
+By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
 For additional EB settings, see <<arcsight-module-config>>. Consuming from a
 secured EB port is not currently available.
 
+NOTE: Logstash, Elasticsearch, and Kibana must be running locally with default ports exposed, namely
+Elasticsearch as "localhost:9200" and Kibana as "localhost:5601". Note that you can also run 
+Elasticsearch, Kibana and Logstash on separate hosts to consume data from ArcSight.
+
 [[arcsight-instructions-eventbroker]]
 ===== Instructions
 
-. {ref}/installing-xpack-es.html[Install X-Pack on Elasticsearch] and then start
-Elasticsearch.
-
-. {kibana-ref}/installing-xpack-kb.html[Install X-Pack on Kibana] and then start
-Kibana.
-
-. {logstash-ref}/installing-xpack-log.html[Install X-Pack on Logstash], which
-includes the Logstash ArcSight module. Then update the Logstash
-<<plugins-inputs-kafka,Kafka input plugin>> to an EB compatible version. In the
-Logstash install directory, run:
+. Update the Logstash <<plugins-inputs-kafka,Kafka input plugin>> to an EB
+compatible version. In the Logstash install directory, run:
 +
 [source,shell]
 -----
```

Comment:
@karenzone we should clean this up this code block as well.

starting 6.3 here, we can remove the install x-pack step since they will need to use the x-pack bundled build. Also, the second step can be removed for 6.0+ since we started bundling a newer version of the kafka input plugin that's backward compatible with older brokers.

as this is a bit of a separate request, let me know if you'd like me to file an issue for it.

Reply:
@acchen97  No need for a separate request.  I've already removed the x-pack reference in response to Jake's comments.  Sounds like all I need to do is remove Step 1 - Update the Kafka input plug-in step entirely.